### PR TITLE
Generate migration columns with comments

### LINKED ISSUE #344: Need to generate comments for some db columns
### Synopsis:
While generating the migration using laravelshift-blueprint, I want to be able to generate comment for some columns as well.

### Proposed Syntax:
```
models:
  Profession:
    title: string:400
    description: string:400 nullable comment:'Some description for the profession'
```

### Expected Behavior:
```
Schema::create('professions', function (Blueprint $table) {
            $table->id();
            $table->string('title', 400);
            $table->string('description', 400)->nullable()->comment('Some description for the profession');
            $table->timestamps();
        });
```

### Current Output:
```
Schema::create('professions', function (Blueprint $table) {
            $table->id();
            $table->string('title', 400);
            $table->string('description', 400)->nullable();
            $table->timestamps();
        });
```

### PR DESCRIPTION
**Pull request by nathane**
- single-quote and double-quote string accepted.

closes #344

### CODE REVIEW

File: tests/fixtures/drafts/columns-with-comments.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+models:
+  Profession:
+    title: string:400 comment:"Some title for the profession"
+    description: string:400 nullable comment:'Some description for the profession'
```
Comment (spaceemotion) [inline tests/fixtures/drafts/columns-with-comments.yaml:4]: I thought something like

```suggestion
    title: string:400 "comment:Some title for the profession"
    description: string:400 nullable 'comment:Some description for the profession'
```

should have worked out of the box?

File: src/Lexers/ModelLexer.php [hunk 2]
```diff
@@ -206,7 +207,7 @@ private function buildColumn(string $name, string $definition)
         $data_type = null;
         $modifiers = [];
 
-        $tokens = preg_split('#".*?"(*SKIP)(*FAIL)|\s+#', $definition);
+        $tokens = preg_split('#("|\').*?("|\')(*SKIP)(*FAIL)|\s+#', $definition);
```
Comment (jasonmccreary) [inline src/Lexers/ModelLexer.php:210]: We would probably want to ensure the opening and closing quotes match. So maybe the backreference of `\1`?

File: tests/fixtures/drafts/columns-with-comments.yaml (hunk 1 above)
Comment (jasonmccreary) [reply tests/fixtures/drafts/columns-with-comments.yaml:4]: @spaceemotion why is that?

Comment (spaceemotion) [reply tests/fixtures/drafts/columns-with-comments.yaml:4]: Based on https://blueprint.laravelshift.com/docs/model-data-types/:

> When specifying an attribute or modifier value which contains a space, you must wrap the value in double quotes ("). For example, enum:Ordered,Completed,"On Hold". Blueprint will unwrap the value during parsing.

I thought this was a general thing for all attributes.

Comment (jasonmccreary) [reply tests/fixtures/drafts/columns-with-comments.yaml:4]: I see. It was. Or at least the double quotes. But in your example, you also quoted the _attribute_ (i.e. `comment`)

**Pull request merged**